forem/forem
# Basic MVP version of data update script page

## What type of PR is this? (check all applicable)


## Description

This is a basic version of a data update script page. It currently lives in `/admin/data_update_scripts`. It can be accessed by someone with admin and tech_admin privileges. 

Since this is the first version, it does not have the following as yet:
- A navbar element 
- Viewing the error when it fails
- Rerunning a script
- I was thinking possibly a link to the actual data script showing the content of the files.
- Possibly some pagination
- Add to documentation

## Related Tickets & Documents
https://github.com/forem/internalEngineering/issues/276

## QA Instructions, Screenshots, Recordings

1. Ensure that you have admin and tech_admin privileges. You can do this finding your user in the console and then add the roles using `your_user.add_role(:admin)` and ` your_user.add_role(:tech_admin)`
2. Go to localhost:3000/data_update_scripts and you should be able to see all the data scripts as follows:

<img width="1679" alt="Screenshot 2021-01-07 at 18 38 59" src="https://user-images.githubusercontent.com/2786819/103918629-b3593e00-5117-11eb-8abf-5f7b2f0af058.png">

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://docs.forem.com/frontend/accessibility)._

## Added tests?

      have not been included_

## Added to documentation?

      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)

## [optional] Are there any post deployment tasks we need to perform?
None

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media.giphy.com/media/IgvC67nz1DgkWqoxPs/giphy.gif)
